game.py

```diff
@@ -382,6 +462,17 @@ def do_expose_event(self, event):
     def _destroy_cb(self, win, event):
         Gtk.main_quit()
 
+    def _new_dot_gameover(self, color):
+        ''' generate a dot of a color color '''
+        self._stroke = color
+        self._fill = color
+        self._svg_width = self._dot_size_gameover
+        self._svg_height = self._dot_size_gameover
+        return svg_str_to_pixbuf(
+            self._header() + \
+            self._circle(self._dot_size_gameover / 2., self._dot_size_gameover / 2.,
+                         self._dot_size_gameover / 2.) + \
+            self._footer())
```

Comment:
Would be more readable if the operator is before the next operand and it's indent should be same as the other operand.